src/test/test_minmax.c

```diff
@@ -80,7 +80,7 @@ const matrix_info files [ ] =
     {                  1,                  1, "matrix_fp32_structure.mtx" } ,
     {          -INFINITY,           INFINITY, "matrix_fp64.mtx" } ,
     {             -32768,              32767, "matrix_int16.mtx" } ,
-    {        -2147483648,         2147483647, "matrix_int32.mtx" } ,
+    {      -2147483648.0,         2147483647, "matrix_int32.mtx" } ,
```

Comment:
Why the decimal?

Reply:
Because otherwise, a compiler complains about the int32 value getting truncated.  It's safer this way and it silences the compiler warning.